virt/virt-2-4-release-notes.adoc

```diff
@@ -0,0 +1,200 @@
+
+//This bug - 1686208- was listed in POST status in Bugzilla:
+* After migration, a virtual machine is assigned a new IP address. However, the
+commands `oc get vmi` and `oc describe vmi` still generate output containing the
+obsolete IP address. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1686208[*BZ#1686208*])
++
+** As a workaround, view the correct IP address by running the following command:
++
+----
+$ oc get pod -o wide
+----
+
+//This defect was closed therefor the content for this defect was deleted:(link:https://bugzilla.redhat.com/show_bug.cgi?id=1743985[*BZ#1743985*])
+//This defect is in POST status [*BZ#1760028*]
+* Live migration fails when nodes have different CPU models. Even in cases where
+nodes have the same physical CPU model, differences introduced by microcode
+updates have the same effect. This is because the default settings trigger
+host CPU passthrough behavior, which is incompatible with live migration.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1760028[*BZ#1760028*])
++
+** As a workaround, set the default CPU model in the `kubevirt-config` ConfigMap,
+as shown in the following example:
++
+[NOTE]
+====
+You must make this change before starting the virtual machines that support
+live migration.
+====
++
+. Open the `kubevirt-config` ConfigMap for editing by running the following command:
++
+----
+$ oc edit configmap kubevirt-config -n openshift-cnv
+----
++
+. Edit the ConfigMap:
++
+[source,yaml]
+----
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+data:
+  default-cpu-model: "<cpu-model>" <1>
+----
+<1> Replace `<cpu-model>` with the actual CPU model value. You can determine this
+value by running `oc describe node <node>` for all nodes and looking at the
+`cpu-model-<name>` labels. Select the CPU model that is present on all of your
+nodes.
+
+* The {VirtProductName} upgrade process occasionally fails due to an interruption
+from the Operator Lifecycle Manager (OLM). This issue is caused by the limitations
+associated with using a declarative API to track the state of {VirtProductName}
+Operators. Enabling automatic updates during
+xref:../virt/install/installing-virt-web.adoc#virt-subscribing-to-the-catalog_installing-virt-web[installation]
+decreases the risk of encountering this issue.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1759612[*BZ#1759612*])
+
+//Jira story for the following content https://issues.redhat.com/browse/CNV-4757. Nelli noted that for CNV 2.5, this can be removed from Known Issues. The defect status, however, is CLOSED.
+* {VirtProductName} cannot reliably identify node drains that are triggered by
+running either `oc adm drain` or `kubectl drain`. Do not run these commands on
+the nodes of any clusters where {VirtProductName} is deployed. The nodes might not
+drain if there are virtual machines running on top of them.
+The current solution is to put nodes into maintenance.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
```

Comment:
Please add a link explaining *how* to put a node into maintenance (e.g https://docs.openshift.com/container-platform/4.2/cnv/cnv_users_guide/cnv-setting-node-maintenance.html )
Without it, this release note puts the reader in a dead-end.

Reply:
and we should not point to a doc bug
need to find something better or remove the link imo

Reply:
This comment was addressed by @aburdenthehand.

Reply:
Added the link.